packages/contracts/tasks/deploy-unirep.ts

```diff
@@ -0,0 +1,209 @@
+import { Contract } from 'ethers'
+import {
```

Comment:
Are we able to use this script outside of a hardhat environment?

Reply:
I think that if users download the repo and deploy contract in the repo they have to use the hardhat environment 
and outside the repo they can write their deploy script, set their environment to deploy.
(and they can get `Unirep.sol` from `@unirep/contracts`, get verifiers/circuits from server or build by themselves)

but not 100% sure about this.
does a `deploy.ts` script in `@unirep/contracts` helpful?